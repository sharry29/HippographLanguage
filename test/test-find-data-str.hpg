int main() {
	graph<int:string> g = [1:"a" -()> 2:"b" <()> 3:"b"; 1 <()- 4];
	graph<int:string> g_new = g.find("b");
	for_node(n : g_new) {
		n.print();
		print("");
	}
}