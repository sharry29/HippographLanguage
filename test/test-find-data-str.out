2:"b"
3:"b"